lib/client.js

```diff
@@ -586,6 +586,9 @@ MqttClient.prototype.end = function (force, cb) {
     that.incomingStore.close(function () {
       that.outgoingStore.close(cb)
     })
+    if (that.deferredReconnect) {
+      that.deferredReconnect()
```

Comment:
as `deferredReconnect`  is only used internally, can you please add a `_`  prefix?

Reply:
OK, I will do it.